docs/docs/08-references/02-config.md

```diff
@@ -12,6 +12,26 @@ to describe the development environment for your blockchain.
 Only a default set of parameters is provided. If more nuanced configuration is
 required, you can add these parameters to the `config.yml` file.
 
+## Validation
+
+The `validation` field is used by Ignite to determine the kind of validation
+of your blockchain. There's is currently 2 supported kinds of validation:
+
+- `sovereign` which is the standard kind of validation where your blockchain
+  has it's own validator set. This is the default value when this field is not
+  present in the config file.
+- `consumer` indicates your blockchain is a consumer chain, in a sense of
+  Replicated Security. That means it doesn't have a validator set, but
+  inherits the one of a provider chain.
+
+While `sovereign` chain is the default validation when you run `ignite scaffold
+chain`, to scaffold a consumer chain, you have to run `ignite scaffold chain
+--consumer`.
+
+This field is, at this time of writing, only used by Ignite at the genesis
+generation step, because the genesis of a sovereign chain and a consumer chain
+are different.
+
```

Comment:
```suggestion
## Validation

Ignite uses the `validation` field to determine the kind of validation
of your blockchain. There are currently two supported kinds of validation:

- `sovereign` which is the standard kind of validation where your blockchain
  has its own validator set. This is the default value when this field is not
  in the config file.
- `consumer` indicates your blockchain is a consumer chain, in the sense of
  Replicated Security. That means it doesn't have a validator set, but
  inherits the one of a provider chain.

While the `sovereign` chain is the default validation when you run the `ignite scaffold
chain`, to scaffold a consumer chain, you have to run `ignite scaffold chain
--consumer`.

This field is, at this time of writing, only used by Ignite at the genesis
generation step, because the genesis of a sovereign chain and a consumer chain
are different.

```